composition/config/crd/bases/composition.google.com_compositions.yaml

```diff
@@ -0,0 +1,231 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.14.0
+  name: compositions.composition.google.com
+spec:
+  group: composition.google.com
+  names:
+    kind: Composition
+    listKind: CompositionList
+    plural: compositions
+    singular: composition
+  scope: Namespaced
```

Comment:
The Composition can be created in any namespace. This could be an admin namespace that the IDP admin chooses it to be. It doesn't limit the Facades to be in the same namespace. Facades can be in any namespace. The controller reconciles compositions in any namespace and created sub-reconciler for each of them. The sub reconcilers looks for Facade from any namespace.

We can discuss this further. Adding an entry in TODO.md

Reply:
What happens when two Compositions use the same Facade as input, with CompositionA in NS1, and CompositionB in NS2?

Then when I create MyFacade in NS3... what do I get? CompositionA or CompositionB (or neither?)